05-mvc-with-rails/error-handling/readme.md

```diff
@@ -3,14 +3,388 @@
 ## Learning Objectives
 
 - Explain the benefits of explicitly handling errors
-- Produce and handle an error in Javascript using the keywords `try`, `catch`, `throw`, and `finally`
 - Produce and handle an error in Ruby using the keywords `begin`, `rescue`, `raise`, and `ensure`
 - Explain the purpose of `flash` in Rails
-- Compare and contrast `flash[:notice]`, `flash[:alert]`, and `flash[:error]`
+- Compare and contrast `flash[:notice]` and `flash[:alert]`
 - List three ActiveRecord methods that trigger validations
-- Compare and contrast the validation helpers `confirmation`, `uniqueness`, `inclusion`, `exclusion`, `length`, `presence`, and `uniqueness`
+- Compare and contrast the validation helpers `confirmation`, `inclusion`, `exclusion`, `length`, `presence`, `uniqueness`, and `numericality`
 - Compare and contrast the validation options `allow_nil`, `allow_blank`, and `on`
 - Create a custom validation on an ActiveRecord model
```

Comment:
friendly reminder to update https://github.com/ga-dc/curriculum/blob/master/05-mvc-with-rails/readme.md if you haven't already